packages/drivers/detox/index.ts

```diff
@@ -3,19 +3,38 @@ import {
   TestingFrameworkDriver,
   TestingFrameworkDriverConfig,
 } from "@wix-pilot/core";
-import detox from "detox";
-import { expect } from "@jest/globals";
 import * as fs from "fs";
 import * as path from "path";
 
 export class DetoxFrameworkDriver implements TestingFrameworkDriver {
+  private _jestExpect: any;
+  readonly detox: any;
+
+  constructor(detox: any) {
+    this._jestExpect = null;
+    this.detox = detox;
```

Comment:
perhaps we can handle jest and detox the same here, I don't see the difference between each problem (both coming from the host package, both injected to the global..)